waveform-django/waveforms/dash_apps/finished_apps/waveform_vis.py

```diff
@@ -204,9 +227,15 @@ def get_user_events(user, project_folder):
     FILE_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir))
     FILE_LOCAL = os.path.join('record-files')
     PROJECT_PATH = os.path.join(FILE_ROOT, FILE_LOCAL)
-
-    if user.is_admin:
+    
+    if user.is_admin and user.practice_status == "ED":
```

Comment:
Just for consistency, let's keep everything single quotes ... there's a couple other places too